formular.html

```diff
@@ -12,12 +13,12 @@ <h1><em>Date Matcher</em></h1>
      
     <form>
       <label>
```

Comment:
Not this way: we want to keep the semantic HTML (<label> for explaining the purpose of input field)
Maybe you achieve the desired effect by pure CSS? 
I know it should be possible.